db/src/main/kotlin/com/trib3/db/jooqext/ResultQueryFlow.kt

```diff
@@ -1,7 +1,9 @@
 package com.trib3.db.jooqext
 
 import kotlinx.coroutines.CancellationException
+import kotlinx.coroutines.CoroutineStart
 import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
```

Comment:
I know Kotlin throws the word Experimental around a lot but it still weirds me out to see it

Reply:
yeah, it basically means "we reserve the right to change this API later" -- kind of like the debian idea of "stable", it doesn't necessarily mean what it sounds like it might!